randutil/crypto.go

```diff
@@ -49,13 +49,16 @@ func CryptoToken(nbytes int) (string, error) {
 	return base64.RawURLEncoding.EncodeToString(b), nil
 }
 
+// Allow mocking of the path for testing purposes.
+var KernelUUIDPath = "/proc/sys/kernel/random/uuid"
```

Comment:
I would not export this, instead I think it would be better to use `kernelUUIDPath` (lower-case) and then `export_test.go` just makes it accessible for the test.